File: examples/performance/array/array.cpp
#include <iostream>
#include <vector>

// g++ array.cpp -o array++ -Ofast

int main()
{
    // Create a vector containing integers
    std::vector<int> arr;

    for(int i=0;i<100000000;i++) {
        arr.push_back(i);
    }
}

File: examples/performance/array/Array.cs
using System;
using System.Collections.Generic;

class Array
{
    static void Main(string[] args)
    {
        List<int> arr = new List<int>();

        for (int i = 0; i < 100_000_000; i++)
        {
            arr.Add(i);
        }
    }
}
